test/e2e/setup.sh

```diff
@@ -25,6 +25,7 @@ export HUB_AGENT_IMAGE="${HUB_AGENT_IMAGE:-hub-agent}"
 export MEMBER_AGENT_IMAGE="${MEMBER_AGENT_IMAGE:-member-agent}"
 export REFRESH_TOKEN_IMAGE="${REFRESH_TOKEN_IMAGE:-refresh-token}"
 export PROPERTY_PROVIDER="${PROPERTY_PROVIDER:-azure}"
+export CLOUD_CONFIG_FILE="${CLOUD_CONFIG_FILE:-"/etc/kubernetes/provider/config.json"}"
```

Comment:
This file path is set fixed in our helm chart. We may not need this configuration at all.